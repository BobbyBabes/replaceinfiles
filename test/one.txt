welcome
welcome
it's me again
I love this world
