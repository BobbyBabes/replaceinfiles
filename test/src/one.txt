hello
hello
it's me again
I love this world
