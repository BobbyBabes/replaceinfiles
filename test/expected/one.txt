goodbye
goodbye
it's me again
I love this earth
